bifrost/app/blog/blogs/from-comfort-zone-to-growth/metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+  "title": "From Code to Customers: An Engineer's Journey into Sales and Distribution",
```

Comment:
"An engineer can learn sales, too" or "How engineers can learn sales and distribution"